qemu/tests/cfg/usb.cfg

```diff
@@ -311,13 +311,15 @@
                     Host_RHEL.m6:
                         # Now no output from monitor for negative test on Host_RHEL.m6
                         usb_reply_msg = ""
-                    usb_reply_msg = "Property 'usb-host.productid' doesn't take value"
+                    usb_reply_msg = "Property 'usb-host.productid' doesn't take value;"
                     usb_reply_msg += "Parameter 'productid' expects an int64 value or range;"
                     usb_reply_msg += "Parameter 'vendorid' expects an int64 value or range;"
                     usb_reply_msg += "productid out of range;"
                     usb_reply_msg += "vendorid out of range;"
                     usb_reply_msg += "Parameter 'productid' expects uint32_t;"
                     usb_reply_msg += "Parameter 'vendorid' expects uint32_t;"
+                    usb_reply_msg += "Parameter 'productid' expects uint64;"
+                    usb_reply_msg += "Parameter 'vendorid' expects uint64"
```

Comment:
Have you tested with the old version ?

Reply:
Yes, it will always be PASS.

Reply:
@ybduan i am puzzled, why both `uint32_t` and `uint64` all can pass ? Could you explain more ?

Reply:
Ok, thanks!